scripts/linux/publishReleasePackages.sh

```diff
@@ -253,8 +253,7 @@ publish_to_github()
     branch_name=${BRANCH_NAME/"refs/heads/"/""}
     echo "Branch Name is $branch_name"
```

Comment:
Above on line 230:
> If SKIP_UPLOAD=false, the script creates a DRAFT github release page on /azure-iotedge

Should it be the other way around?

Reply:
Good catch! thanks!